cpp-client/tools/aiaa/aiaa-fix-polygon.cpp

```diff
@@ -32,15 +32,33 @@
 #include <chrono>
 
 int main(int argc, char **argv) {
+  if (1 == 0) {
```

Comment:
should this stays in here,
or this is a test?

Reply:
nope.. fixed